pyOCD/coresight/ap.py

```diff
@@ -337,8 +337,10 @@ def _handle_error(self, error, num):
 
 class AHB_AP(MEM_AP):
     def init(self, bus_accessible=True):
-        super(AHB_AP, self).init(bus_accessible)
+        # Init super first with bus access disabled, so it doesn't read the ROM table.
+        super(AHB_AP, self).init(False)
```

Comment:
Moving renaming and moving the MEM_AP_IDR_TO_WRAP_SIZE sounds good.

By AP vs MEM_AP and ROM table, meant to convey that coresight does not define a ROM table for a generic AP, so I was unsure why code to read the ROM table was there. I believe only the IDR register is defined for an AP. I think its the MEM-AP which first defines the register containing the rom table address - "BASE, Debug Base Address register". That is why I mentioned the MDM-AP which is not a type of MEM-AP and has no concept of a ROM table.